pom.xml

```diff
@@ -124,6 +125,7 @@
             </plugin>
             <plugin>
                 <artifactId>maven-source-plugin</artifactId>
+                <version>2.4</version>

```

Comment:
Not sure where I saw the note now. VersionEye did report that our plugin versions were out-of-date. I suppose this will be easier to keep updated once we have the parent pom in-place.